setup.py

```diff
@@ -116,6 +116,7 @@ def run(self):
     "MySQL-python>=1.2.5",
     "numpy>=1.7",
     "nydus>=0.10.8",
+    "oauthlib==0.7.2",
```

Comment:
it's not permitted by `requirements.py`.
Maybe if you set major and minor version like `oauthlib>=0.7.2<1.0`. 
But, what's the difference? This is an "hack" to require a specific (old) version and in future we need to fix dependencies anyway.